di-ipv-queue-stub/enqueue-event/src/handlers/enqueueEvent.ts

```diff
@@ -12,50 +12,57 @@ import {
     CreateQueueCommand,
     CreateQueueCommandInput, AddPermissionCommandInput, CreateQueueCommandOutput, AddPermissionCommandOutput
 } from "@aws-sdk/client-sqs";
+import { buildSignedJwt } from 'di-stub-oauth-client';
+import type { RequestPayload } from '../types';
+import type { SignedJwtParams } from 'di-stub-oauth-client';
 export const handler: Handler = async (
     event: any,
     context:any
 ) => {
     console.log("event:");
     console.log(event);
     let aws_account_id = JSON.stringify(context.invokedFunctionArn).split(':')[4];
-    let body = JSON.parse(event.body);
+    const body: RequestPayload = JSON.parse(event.body);
     console.log("body:");
     console.log(body);
-    let message = body.queueEvent;
+
     const sqsConfig = {
         region: "eu-west-2"
     }
     const sqsClient = new SQSClient(sqsConfig);
-    let getMessageQueueUrlInput:GetQueueUrlCommandInput = {
+    let getMessageQueueUrlInput: GetQueueUrlCommandInput = {
         QueueName: body.queueName
     }
     let getMessageQueueUrlCommand = new GetQueueUrlCommand(getMessageQueueUrlInput);
     let getMessageQueueUrlOutput: GetQueueUrlCommandOutput;
-    let queueUrl: string ;
+    let queueUrl: string;
     try {
         getMessageQueueUrlOutput = await sqsClient.send(getMessageQueueUrlCommand);
         console.log("getMessageQueueUrlOutput:");
         console.log(getMessageQueueUrlOutput);
         queueUrl = getMessageQueueUrlOutput.QueueUrl;
-    }catch(error) {
+    } catch (error) {
         console.log("in Catch");
         console.log(error);
-        if ( error instanceof QueueDoesNotExist) {
+        if (error instanceof QueueDoesNotExist) {
             //queue doesn't exist - let's make one
             console.log("missing queue");
-            let createQueueCommandInput:CreateQueueCommandInput = {
+            let redrivePolicy = {
+                deadLetterTargetArn: "arn:aws:sqs:eu-west-2:" + aws_account_id + ":stubQueue_F2FDLQ",
```

Comment:
Minor but another way of doing this is `` deadLetterTargetArn: `arn:aws:sqs:eu-west-2:${aws_account_id}:stubQueue_F2FDLQ` `` which is arguably cleaner.